app/views/sign_up/completions/_show_sp.html.erb

```diff
@@ -13,8 +13,8 @@
 </p>
 <p>
   <div class="center">
-    <%= button_to t('sign_up.agree_and_continue'),
-                  sign_up_completed_path,
-                  class: 'btn btn-primary btn-wide' %>
+    <%= validated_form_for(:idv_form, url: sign_up_completed_path) do %>
+      <%= submit_tag t('sign_up.agree_and_continue'), class: 'btn btn-primary btn-wide' %>
+    <% end %>
```

Comment:
I admit don't fully understand if and how symbols like `:idv_form` are resolved, but the IDV term feels a bit out of place here. Is it something where it doesn't really matter for how it's used here? Can it be omitted?

Reply:
I think it's used for the input names, so they become `<input name=user[email]>`, etc.

in this case it doesn't matter, and we could call it `:form` or something

Reply:
I am not too familiar with this method or how it evolved.  I did add the hook into the js to prevent the double submit a while ago.  It looks like normally it's supposed to be a form object but when there are no inputs (like just a submit button) the form object is not necessary.  I just did a quick search and followed the convention of the other screens late in the idv flow and fed it that symbol.  I assume that would update the naming of any of the inputs if there were some.

Reply:
Broadly I think this should be a form submission, maybe even a `POST` since it changes state on the backend 🙂

Reply:
Yep.  It is a post.

Reply:
I think the form defaults to GET unless we do `validated_form_for(:idv_form, url: url, method: 'POST'`?